inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/AnnotationQueueOverviewDataProvider.java

```diff
@@ -78,12 +81,34 @@ public TableContentProvider(
             private static final long serialVersionUID = -3938543310389673460L;
 
             @Override
-            protected List<SourceDocument> load() {
+            protected List<SourceDocument> load()
+            {
+                return data;
+            }
+        };
+    }
+
+    public AnnotationQueueOverviewDataProvider(
+        List<AnnotationDocument> aAlAnnotationDocuments,
+        List<SourceDocument> aData, DocumentService aDocumentService, EntityManager aEntityManager)
+    {
+        entityManager = aEntityManager;
+        allAnnotationDocuments = aAlAnnotationDocuments;
+        data = aData;
+        documentService = aDocumentService;
+        headers = new ArrayList<>();
+        model = new LoadableDetachableModel<List<SourceDocument>>() {
```

Comment:
LoadableDetachableModel is used to access fresh data from some data source on every request. However, your "data" object is not a dynamic data source or from a non-serializable source. It is always the same object. What's the point of using LoadableDetachableModel here?